setup.cfg

```diff
@@ -1,2 +0,0 @@
-[bdist_wheel]
-universal=1
```

Comment:
does this have any consequences for existing usage?

Reply:
I have moved this parameter in the `pyproject.toml` file. I wanted to limit the number of configuration files. I did it in advance as I hope that, in a future PR, the `setup.py` file will be removed from the repository (to be replaced with additional data `pyproject.toml`)